README.md

```diff
@@ -71,30 +100,47 @@ depending on how the build was triggered from GitHub.
 
 Packer will use your [standard AWS
 environment](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)
-to build the image.
+to build the image, however you will need to set up one profile for
+the previously-created build user and another profile to assume the
+associated `EC2AMICreate` role.  You will need the `aws_access_key_id`
+and `aws_secret_access_key` that you set as GitHub secrets earlier.
+
+Add the following blocks to your AWS credentials file (be sure to
+replace the dummy account ID in the `role_arn` with your own):
+
+```console
+[test-skeleton-packer-cool]
+aws_access_key_id = AKIAXXXXXXXXXXXXXXXX
+aws_secret_access_key = XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+[cool-images-ec2amicreate-skeleton-packer-cool]
+role_arn = arn:aws:iam::111111111111:role/EC2AMICreate-test-skeleton-packer-cool
+source_profile = test-skeleton-packer-cool
```

Comment:
This section here has some `skeleton-packer-cool` text that should be changed to `kali-packer`.

Reply:
See commit d4c9948.